cclib/parser/turbomoleparser.py

```diff
@@ -48,6 +48,9 @@ class Turbomole(logfileparser.Logfile):
 
     def __init__(self, *args, **kwargs):
         super(Turbomole, self).__init__(logname="Turbomole", *args, **kwargs)
+        
+        # Flag for whether this calc is DFT.
+        self.DFT = False

```

Comment:
Some people hate this, but this is where I like a variant of Hungarian notation, or like how Lisp does it:
```suggestion
        self.is_DFT = False
```
What do you think?

Reply:
I'm not against this personally. I based this naming of the 'convention' (if that's the right term) used in the Gaussian Parser, which is not to include the 'is_' prefix. eg (coupledcuster, oniom, bomd etc):
```
def before_parsing(self):

        # Extract only well-formed numbers in scientific notation.
        self.re_scinot = re.compile(r'(\w*)=\s*(-?\d\.\d{2}D[+-]\d{2})')
        # Extract only well-formed numbers in traditional
        # floating-point format.
        self.re_float = re.compile(r'(\w*-?\w*)=\s*(-?\d+\.\d{10,})')

        # Flag for identifying Coupled Cluster runs.
        self.coupledcluster = False

        # Fragment number for counterpoise or fragment guess calculations
        # (normally zero).
        self.counterpoise = 0

        # Flag for identifying ONIOM calculations.
        self.oniom = False

        # Flag for identifying BOMD calculations.
        # These calculations have a back-integration algorithm so that not all
        # geometries should be kept.
        # We also add a "time" attribute to the parser.
        self.BOMD = False

        # Do we have high-precision polarizabilities printed from a
        # dedicated `polar` job? If so, avoid duplicate parsing.
        self.hp_polarizabilities = False
```
Happy to change tho?

Reply:
Yes, let's switch. The Gaussian parser is one of the crustier ones.